src/1_Iterations/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Given a positive integer N, Computes the length of its longest binary gap. 
+ * Returns 0 if N doesn't contain a binary gap
+ * @param {number} N 
+ */
+function solution(N) {
+  if (!Number.isInteger(N) || N < 0)
+    throw new Error("N must be a positive integer")
+
+  const binary = N.toString(2)
+
+  let aug = binary
+  do {
+    aug = aug
```

Comment:
Implement this without regex. Let's be as crude as possible. Just use variables and `Iterations`.